oz/sqlalchemy/__init__.py

```diff
@@ -56,6 +98,24 @@ def engine(connection_string=None):
             kwargs["max_overflow"] = oz.settings["db_max_overflow"]
         if oz.settings["db_pool_timeout"]:
             kwargs["pool_timeout"] = oz.settings["db_pool_timeout"]
+        if oz.settings["db_pool_pre_ping"]:
+            major, minor, *patch = sa_version.split(".")
```

Comment:
nit: might be nice to abstract this version checking logic to a return bool that you can then act on

Reply:
Might be nice to destroy oz. I'm making no effort to do anything other than make this library usable for immediate needs. We are getting any new iteration of an application off of it.